main.go

```diff
@@ -86,10 +86,20 @@ func parseFlags(program string, args []string) (c *config, output string, err er
 		return nil, buf.String(), err
 	}
 
+	if conf.gf == "" && conf.db == "" {
+		flags.PrintDefaults()
+		return nil, buf.String(), errors.New(
+			"-gf is required and -db can not be an emptry string",
+		)
+	}
```

Comment:
Is this to enable present a complete picture of the argument issues to the user? I think it'd be OK to simply not do this, e.g. the first error is shown, and the user tries again. However if you did want to give a complete picture of the errors, you could collect them in a []string and then join that into an error -- having a specific case where both args are missing could get hard to maintain if another arg was added.

Reply:
Thats right; I get annoyed when there are multiple unconnected things that need to be fixed, but a program only tells me about one of them, and, 'fixing' that seemed easy here. I was originally going to gather them up as you suggest, but that started making things a little more complicated. I agree this will need to be changed if more flags are added, but I think I'm ok with leaving that as a potential future problem.